src/translations/mapCollection.js

```diff
@@ -0,0 +1,114 @@
+import { isUndefined, isDefined } from '../helpers/mixed';
+import { mixin } from '../helpers/object';
+import localHooks from '../mixins/localHooks';
+
+// Counter for checking if there is a memory leak.
+let registeredMaps = 0;
+
+class MapCollection {
```

Comment:
Missing jsdoc about the purpose of this class

Reply:
The description that you wrote can be deferred directly from the class name. I would expect an explanation about the purpose of this class, e.g.:

> Allows performing batch operations on all maps that have been registered in the collection.

Reply:
As for `IndexMapper`, `MapCollection` doesn't have TS definition. Maybe this class is hidden so deeply that no TS definition is necessary for developer who writing in TS?